sdk/communication/azure-communication-callingserver/pom.xml

```diff
@@ -54,7 +54,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+      <version>1.0.2</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```

Comment:
```suggestion
      <version>1.0.2</version> <!-- {x-version-update;com.azure:azure-communication-common;dependency} -->
```

Reply:
Change common project version back to 1.1.0-beta.1